src/zmalloc.c

```diff
@@ -626,9 +626,54 @@ size_t zmalloc_get_rss(void) {
 
 #if defined(USE_JEMALLOC)
 
-int zmalloc_get_allocator_info(size_t *allocated,
-                               size_t *active,
-                               size_t *resident) {
+#include <assert.h>
+
+#define STRINGIFY_(x) #x
+#define STRINGIFY(x) STRINGIFY_(x)
+
+/* Compute the total memory wasted in fragmentation of inside small arena bins.
+ * Done by summing the memory in unused regs in all slabs of all small bins. */
+size_t zmalloc_get_frag_smallbins(void) {
```

Comment:
with simple benchmarks, this method does not cause significant performance regression. 

unstable:
```
30        Threads
50        Connections per thread
10000     Requests per client

AGGREGATED AVERAGE RESULTS (3 runs)
============================================================================================================================
Type         Ops/sec     Hits/sec   Misses/sec    Avg. Latency     p50 Latency     p99 Latency   p99.9 Latency       KB/sec 
----------------------------------------------------------------------------------------------------------------------------
Sets        29869.63          ---          ---        45.40777        46.07900        47.61500        92.15900      2300.48 
Gets       298368.11         0.00    298368.11        45.39571        46.07900        47.61500        92.15900     11622.73 
Waits           0.00          ---          ---             ---             ---             ---             ---          --- 
Totals     328237.74         0.00    298368.11        45.39681        46.07900        47.61500        92.15900     13923.21 
```

this PR:
```
30        Threads
50        Connections per thread
10000     Requests per client

AGGREGATED AVERAGE RESULTS (3 runs)
============================================================================================================================
Type         Ops/sec     Hits/sec   Misses/sec    Avg. Latency     p50 Latency     p99 Latency   p99.9 Latency       KB/sec 
----------------------------------------------------------------------------------------------------------------------------
Sets        29721.66          ---          ---        45.75202        46.07900        47.87100        92.15900      2289.09 
Gets       296890.02         0.00    296890.02        45.74719        46.07900        47.87100        92.15900     11565.15 
Waits           0.00          ---          ---             ---             ---             ---             ---          --- 
Totals     326611.69         0.00    296890.02        45.74763        46.07900        47.87100        92.15900     13854.24
```